### PR TITLE
Add ✔ to readme

### PR DESCRIPTION
**Pull request by Fr-Dae**
🗷✘✗✖✕

✅✔✓☑🗹🗸✔✓

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,25 +104,25 @@ Bleachbit is a master at finding and removing temporary files, while Czkawka onl
...
-|          Bad ID          |             |     •      |                   |             |
-|  Non stripped binaries   |             |     •      |                   |             |
-|   Redundant whitespace   |             |     •      |                   |             |
-|    Overwriting files     |             |     •      |                   |      •      |
-|    Multiple languages    |      •      |     •      |         •         |      •      |
-|      Cache support       |      •      |            |         •         |             |
+|     Duplicate finder     |     ✔     |    ✔     |        ✔        |             |
+|       Empty files        |     ✔     |    ✔     |                   |             |
+|      Empty folders       |     ✔     |    ✔     |                   |             |
+|     Temporary files      |     ✔     |    ✔     |                   |     ✔     |
```
Comment (qarmin) [inline README.md]: Columns needs to have same width as before(constant)

File: README.md [hunk 2]
```diff
@@ -28,7 +28,8 @@
 ![Czkawka](https://user-images.githubusercontent.com/41945903/145280350-506f7e94-4db0-4de7-a68d-6e7c26bbd2bf.gif)
 
 ## Supported OS
-Linux - Ubuntu 22.04+, Fedora 36+, Alpine Linux 3.16+, Debian 12+ and a lot of more   
+Linux - Ubuntu 22.04+, Fedora 36+, Alpine Linux 3.16+, Debian 12+ and a lot of more   ![image](https://user-images.githubusercontent.com/23202421/218338157-ece73ca7-489c-4204-9a6b-da9583063ae5.png)
```
Comment (qarmin) [inline README.md]: Hmm.......

**Comment by alexislefebvre** [comment]
Why the title and description only contain emojis? It looks like troll or spam.

File: README.md (hunk 1 above)
Comment (Fr-Dae) [reply README.md]: it has no influence or importance.
look at the final file 
https://github.com/qarmin/czkawka/blob/2a0be42eaef7928f2ee5d3bd3e0edb4b7b17d559/README.md

**Comment by Fr-Dae** [comment]
https://github.com/qarmin/czkawka/blob/2a0be42eaef7928f2ee5d3bd3e0edb4b7b17d559/README.md
this has no influence or importance.
look at the final file

**Comment by Fr-Dae** [comment]
> Why the title and description only contain emojis? It looks like troll or spam.

why not ?
you prefer "I replaced points with checks".
Dude, that's a boring as hell title.

File: README.md (hunk 1 above)
Comment (Serial-ATA) [reply README.md]: @Fr-Dae It's not a matter of the final render, the file should have consistent style.

File: README.md (hunk 2 above)
Comment (qarmin) [reply README.md]: This Hmm... meant Hell No!  😄️

Screenshot from 2023-02-20 07-33-00

File: README.md (hunk 1 above)
Comment (Fr-Dae) [reply README.md]: github's display management does not take into account the number of spaces you leave before or after the |
It makes no difference whatsoever.

If you want to modify the readme, go ahead.
In any case when I edit it on my side all is correctly aligned.
Do you use a unicode font?

File: README.md (hunk 2 above)
Comment (Fr-Dae) [reply README.md]: ha! indeed this was not intended,

Comment (Fr-Dae) [reply README.md]: > This Hmm... meant Hell No! smile
> 
> Screenshot from 2023-02-20 07-33-00

I corrected the extra check, probably a miss click
and I added a few spaces, so the other neurotic doesn't have a heart attack :D

**Comment by alexislefebvre** [comment]
> why not ? you prefer "I replaced points with checks". Dude, that's a boring as hell title.

I'm not sure to understand your point here. Do you maintain some open sources project? Do you prefer issues and PRs with titles that explain the changes, or emojis that add no information?

**Pull request merged**